config.go

```diff
@@ -30,50 +30,63 @@ func getConfig() *types.Configuration {
 	v.SetDefault("ListenAddress", "")
 	v.SetDefault("ListenPort", 2801)
 	v.SetDefault("Debug", false)
-	v.SetDefault("CheckCert", true)
+	v.SetDefault("MutualTlsFilesPath", "/certs")
```

Comment:
```suggestion
	v.SetDefault("MutualTlsFilesPath", "/etc/certs")
```

I prefer `/etc/certs` because it's more convenient when :falcosidekick: runs outside Docker